packages/gitgraph-react/src/Commit.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
+import {
+  GitgraphCore,
+  Commit,
+  Mode,
+  Coordinate,
+} from "@gitgraph/core";
+import { CommitElement, ReactSvgElement } from "./types";
+import { Dot } from "./Dot";
+import { Tooltip } from "./Tooltip";
+import { Arrow } from "./Arrow";
+import { Message } from "./Message";
+import { Tag } from "./Tag";
+import { BranchLabel } from "./BranchLabel";
+
+interface CommitsProps {
+  commits: Array<Commit<ReactSvgElement>>;
+  commit: Commit<ReactSvgElement>;
+  currentCommitOver: Commit<ReactSvgElement> | null;
+  gitgraph: GitgraphCore<ReactSvgElement>;
+  initCommitElements: (commit: Commit<ReactSvgElement>) => void;
+  commitsElements: {
+    [commitHash: string]: CommitElement;
+  };
+  getWithCommitOffset: (props: any) => Coordinate;
+  setTooltip: (val: React.ReactElement<SVGGElement> | null) => void;
+  setCurrentCommitOver: (val: Commit<ReactSvgElement> | null) => void;
+}
+
+export class CommitComp extends React.Component<CommitsProps, {}> {
```

Comment:
What is `Comp` referring to? Components?

What do you think about naming it `Commit` for consistency with the file name and other components?

Reply:
Yup `Comp` is just short for "Component". I can totally rename it to `Commit`, I was just having some issues with auto-importing in my IDE (it was trying to import `Commit` from other internal packages), but I'll make that change shortly